api/src/main/resources/org/openmrs/api/db/hibernate/LoginCredential.hbm.xml

```diff
@@ -28,6 +28,8 @@
 		<property name="salt" type="java.lang.String" column="salt" length="128" />
 		<property name="secretQuestion" type="java.lang.String" column="secret_question" length="255" />
 		<property name="secretAnswer" type="java.lang.String" column="secret_answer" length="255" />
+		<property name="activationKey" type="java.lang.String" column="activation_key" length="150" />
```

Comment:
Why did you change the length to 150?

Reply:
The hashing generated a long string which could not be accomodated by the innitial 50 and also i increased it to 128 as it done with the password since they used thesame hasing algorithm and it still didn't hold because i append the timestamp to it so i had to increase it.

Reply:
Then I guess have it as 255 which is the standard

Reply:
ok

Reply:
change the length